mapping/chamberInfo.py

```diff
@@ -93,7 +92,7 @@
     #        #Set the Latency - 10x10 PMT on R&D Setup
     #        "CFG_LATENCY":98,
     #        #Correct the bug in the shaper
-    #        "CFG_PT":7,
+    #        "CFG_PT":3,
```

Comment:
revert.

Reply:
And this is 873af002f2d2ee894accc6be1a0c36a35e152ee4